modules/client/src/client-common/constants.ts

```diff
@@ -0,0 +1,8 @@
+import { ProposalMetadata } from "./interfaces/plugin";
+
+export const UNSUPPORTED_PROPOSAL_METADATA_LINK: ProposalMetadata = {
```

Comment:
I strongly suggest we have another template for cases where metadata cannot be fetched despite the link being valid. Otherwise one proposal with unpinned metadata would make everything fail